pages/content/amp-dev/documentation/guides-and-tutorials/develop/custom-javascript.md

```diff
@@ -3,120 +3,141 @@ $title: Use custom JavaScript in AMP pages
+
+## amp-script - or amp-bind and amp-list?
+
+For a new AMP developer who's comfortable with JavaScript, it may seem easier to always use `amp-script` than to learn to use `amp-bind` and `amp-list`. But the complete AMP developer is likely to want to know and use all of these components.
+
+`amp-bind` is generally more straightforward for basic interactions, where its tight integration into HTML tags is compelling. In this example, pressing a button changes a text fragment. AMP's data binding makes this straightforward and easy to read:
+
+```html
+<p [text]="name">Rajesh</p>
+<button on="tap:AMP.setState({name: 'Priya'})">I am Priya</button>
+```
+
+Similarly, when using an API whose output you control, you may be able to implement business logic on the server, and you may be able to format the data it outputs so that it fits smoothly into an `amp-mustache` template. `amp-list` is a good fit in such cases.
+
+`amp-bind` also provides a straightforward mechanism to communicate between AMP components. In this example, tapping on an image in an `<amp-selector>` sets the state variable `selectedSlide` to `0`, which in turn makes an `<amp-carousel>` move to its first slide.
 
 ```html
-<body>  
-  <p>Hello!</p>
-  <div>
-    <amp-script layout="container" src="customjs.js">
-    </amp-script>
-  </div>
-</body>
+<amp-carousel slide="selectedSlide">
+...
+</amp-carousel>
+
+<amp-selector>
+  <amp-img on="tap:AMP.setState({selectedSlide: 0})"/>
+</amp-selector>
 ```
 
-It would result in this:
+Traditional interactive AMP components may also be more suitable for interactions that span large sections of a webpage - as you may not wish to wrap so much of the DOM in an `<amp-script>`. Generally speaking, `amp-list` and `amp-bind` are tightly integrated with the rest of AMP, and it's straightforward to sprinkle these throughout a webpage.
+
+On the other hand, on pages that involve more complex state variables or multiple interactions, `amp-script` will likely be simpler and more maintainable. Take this example, from the [AMP Camp e-commerce demo site](https://camp.samples.amp.dev/product-details?categoryId=women-shirts&productId=79121):
```

Comment:
Cut "on the other hand" for translation reasons.

Reply:
I wanted a longer clause here to demarcate a change from pro-amp-bind to pro-amp-script. But I am shortening to the less anthropomorphic "However"